src/pages/billing/Project.js

```diff
@@ -519,6 +523,14 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
 
   const isGcpProject = billingProject.cloudPlatform === cloudProviders.gcp.label
 
+  const json = Utils.maybeParseJSON(errorMessage)
+  const jsonFrame = {
```

Comment:
Since this is only used once, could it be inlined instead of stored in a variable?